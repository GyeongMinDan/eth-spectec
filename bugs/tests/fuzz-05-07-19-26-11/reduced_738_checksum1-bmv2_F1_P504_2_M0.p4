match_kind {
  selector
}
struct standard_metadata_t {
}
enum CounterType {
  packets,
};
enum MeterType {
  packets,
}
extern meter {
  meter(bit<32> size, MeterType type = {#});
}
enum HashAlgorithm {
  crc32,
};
struct headers {
}
struct metadata {
}
error {
  IPv4ChecksumError
}
control cEgress(inout headers hdr, inout metadata meta, inout standard_metadata_t stdmeta)() {
  apply {}
}
