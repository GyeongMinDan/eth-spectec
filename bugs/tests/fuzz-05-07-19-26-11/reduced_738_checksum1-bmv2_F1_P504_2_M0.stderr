hits-likely-05-07-19-26-11/reduced_738_checksum1-bmv2_F1_P504_2_M0.p4(25): [--Wwarn=unused] warning: Control cEgress is not used; removing
control cEgress(inout headers hdr, inout metadata meta, inout standard_metadata_t stdmeta)() {
        ^^^^^^^
[--Wwarn=missing] warning: Program does not contain a `main' module
